lib/libspl/include/assert.h

```diff
@@ -54,26 +55,29 @@ libspl_assertf(const char *file, const char *func, int line, char *format, ...)
 	va_end(args);
 	abort();
 }
+/* LCOV_EXCL_STOP */
 
 #ifdef verify
 #undef verify
 #endif
 
 #define	VERIFY(cond)							\
-	(void) ((!(cond)) &&						\
-	    libspl_assert(#cond, __FILE__, __FUNCTION__, __LINE__))
+	(void) (("LCOV_EXCL_LINE") && ((!(cond)) &&			\
```

Comment:
I think it would be preferable to use `LCOV_EXCL_BR_LINE` for all these conditionals if does work as expected.

Reply:
OK.

If we plan to move forward with the buildbot PR to expose the HTML pages, then I can use that to try out both approaches; and we can compare the HTML report with Codecov for both.

If not, I'll do that same testing locally and create the HTML reports for both, so we can evaluate the results of both approaches.

Reply:
Let's go forward with the buildbot PR.